vendor/src/github.com/docker/notary/tuf/store/httpstore.go

```diff
@@ -33,6 +45,19 @@ func (err ErrMaliciousServer) Error() string {
 	return "Trust server returned a bad response."
 }
 
+// ErrInvalidOperation indicates that the server returned a 400 response and
+// propagate any body we received.
+type ErrInvalidOperation struct {
+	msg string
+}
+
+func (err ErrInvalidOperation) Error() string {
+	if err.msg != "" {
+		return fmt.Sprintf("Trust server rejected operation: %s", err.msg)
```

Comment:
Minor nit: it's customary for error messages to start with a lowercase letter, because error messages might be chained together (as is being done here).